### PR TITLE
COMMERCE-4658 fix shipping tax configuration category position

### PR DESCRIPTION
**Pull request by igorbeslic**
Please review

### CODE REVIEW

**Comment by igorbeslic** [comment]
ci:test:sf

**Comment by marco-leo** [comment]
ci:test:relevant

File: modules/apps/commerce/commerce-tax-service/build.gradle [hunk 1]
```diff
@@ -10,6 +10,7 @@ dependencies {
 	compileOnly group: "javax.servlet", name: "javax.servlet-api", version: "3.0.1"
 	compileOnly group: "org.osgi", name: "org.osgi.core", version: "5.0.0"
 	compileOnly group: "org.osgi", name: "org.osgi.service.component.annotations", version: "1.3.0"
+	compileOnly project(":apps:configuration-admin:configuration-admin-api")
```
Comment (dsitu) [inline modules/apps/commerce/commerce-tax-service/build.gradle:13]: @igorbeslic this should be sorted.

**Review by dsitu — CHANGES_REQUESTED** [review]

**Comment by marco-leo** [comment]
ci:test:relevant

File: modules/apps/commerce/commerce-tax-service/build.gradle (hunk 1 above)
Comment (igorbeslic) [reply modules/apps/commerce/commerce-tax-service/build.gradle]: Thank you Danny!

**Comment by igorbeslic** [comment]
ci:test:sf
ci:test:relevant

**Comment by igorbeslic** [comment]
ci:test:relevant

**Comment by igorbeslic** [comment]
ci:forward

**Pull request closed without merging**